internal/util/kafka.go

```diff
@@ -19,16 +25,12 @@ func (k kafkautil) NewReader(topic string) *kafka.Reader {
 	var dialer *kafka.Dialer
 
 	if config.DefaultConfig.KafkaUsername != "" && config.DefaultConfig.KafkaPassword != "" {
```

Comment:
```suggestion
	if config.DefaultConfig.KafkaCAPath != "" {
```
In my local setup I am using kafka with custom ssl certs but password less authentication. So in my case `KafkaUsername` and `KafkaPassword` is everytime set to null. I don't think this will be the case with AWS MSK but still we can cover this corner case.